src/drawer/pages/ServiceDetail/ServiceDetail.tsx

```diff
@@ -17,12 +28,35 @@ import {
   StyledServiceInfoContainer,
   StyledServiceTitleText,
   StyledThumbnailImage,
+  StyledDescription,
+  StyledDescriptionPart,
+  StyledDescriptionSection,
+  StyledMoreProductSection,
+  StyledProductImage,
+  StyledCarousel,
+  StyledSubtitle,
+  StyledCarouselButton,
+  StyledLowerSection,
 } from './ServiceDetail.style';
 
 export const ServiceDetail = () => {
   const { serviceId } = useParams();
   const theme = useTheme();
 
+  const scrollRef = useRef<HTMLDivElement | null>(null);
+
+  const handleCarouselClick = (event: React.MouseEvent) => {
+    const target = event.target as HTMLInputElement;
+    const scrollAmount = 450;
+
+    if (scrollRef.current) {
+      const x = scrollRef.current!.scrollLeft;
+      const scrollDirection = target.name === 'left' ? -scrollAmount : scrollAmount;
+
+      scrollRef.current.scrollTo(x + scrollDirection, 0);
```

Comment:
버튼 클릭했을 때의 스크롤 동작이 살짝 어색해 보이는데 `behavior: 'smooth'` 추가 해주면 어떨까요,,

```suggestion
scrollRef.current.scrollTo({ left: x + scrollDirection, behavior: 'smooth' });
```

Reply:
멋져요,,,,
스타일 코드니까 style.ts에 추가할게요 >.<